tokens/src/core/global/elevation.json

```diff
@@ -1,19 +1,61 @@
 {
   "elevation": {
+    "$type": "ratio",
```

Comment:
[sanity check] Does `ratio` make sense as the type (see [DTCG format spec](https://tr.designtokens.org/format/#additional-types)'s description of the `ratio` type)? Perhaps we want this to be `"$type": "number"` since it's really just a number ([docs](https://tr.designtokens.org/format/#number))?

Reply:
[inform] Implemented in follow-up PR: https://github.com/openedx/paragon/pull/3286